docker/Dockerfile

```diff
@@ -54,25 +89,33 @@ RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/inst
 RUN poetry config virtualenvs.create false && \
     poetry config installer.parallel false
 
-# Keep the project source code from NAUTOBOT_ROOT
+# Install (non-development) Python dependencies of Nautobot
 COPY pyproject.toml poetry.lock /source/
 COPY examples /source/examples
 WORKDIR /source
+# pyuwsgi doesn't support ssl so we build it from source
+# https://github.com/nautobot/nautobot/issues/193
+RUN pip install --no-cache-dir --no-binary=pyuwsgi pyuwsgi==2.0.20 && \
```

Comment:
Can we make the pyuwsgi a buildarg so we don't hard code it here

Reply:
Hardcode it into `tasks.py` instead?

Reply:
Figured out a relatively simple regex to parse it out of pyproject.toml.

Reply:
shouldn't need reges, should be able to use the toml lib to grab it

Reply:
ISTR we had some issues in the past (possibly with a different repo entirely) where adding a dependency on `toml` in `tasks.py` caused some chicken-and-egg problems with initially getting a dev environment set up. I may be misremembering.